<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductoProveedorControl</class>
 <widget class="QWidget" name="ProductoProveedorControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>171</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Proveedor:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbProveedor">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Referencia:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtReferencia">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Precio de compra:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CRelaxedDoubleSpinBox" name="numPrecioCompra">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="suffix">
        <string> €</string>
       </property>
       <property name="maximum">
        <double>99999999.989999994635582</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnQuitarProveedor">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="window-close">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Precio con IVA:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="txtPrecioConIVA">
       <property name="styleSheet">
        <string notr="true">margin-right: 10px;</string>
       </property>
       <property name="text">
        <string>0,00 €</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Porcentaje ganancia:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CRelaxedDoubleSpinBox" name="numPorcentajeGanancia">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="suffix">
        <string> %</string>
       </property>
       <property name="maximum">
        <double>9999.989999999999782</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Precio recomendado:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="txtPrecioRecomendado">
       <property name="styleSheet">
        <string notr="true">margin-right: 10px;</string>
       </property>
       <property name="text">
        <string>0,00 €</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnEstablecerPrecioVenta">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Establecer precio de venta</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Observaciones:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="txtObservaciones">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>60</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRelaxedDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>crelaxeddoublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btnEstablecerPrecioVenta</sender>
   <signal>clicked()</signal>
   <receiver>ProductoProveedorControl</receiver>
   <slot>btnSetPrecioVentaClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>633</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnQuitarProveedor</sender>
   <signal>clicked()</signal>
   <receiver>ProductoProveedorControl</receiver>
   <slot>btnQuitarProveedorClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>699</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btnSetPrecioVentaClicked()</slot>
  <slot>btnQuitarProveedorClicked()</slot>
 </slots>
</ui>
